core/src/main/java/greencity/validators/ManualPaymentRequestValidator.java

```diff
@@ -13,23 +13,19 @@
 import static greencity.constant.ValidationConstant.PAYMENT_DATE_FORMAT_IS_NOT_VALID_MESSAGE;
 import static greencity.constant.ValidationConstant.PAYMENT_DATE_IS_AFTER_CURRENT_DATE_MESSAGE;
 import static greencity.constant.ValidationConstant.PAYMENT_DATE_IS_BEFORE_ORDER_CREATION_MESSAGE;
-import static greencity.constant.ValidationConstant.PAYMENT_DATE_IS_NULL_MESSAGE;
 
 @RequiredArgsConstructor
 public class ManualPaymentRequestValidator
     implements ConstraintValidator<ValidManualPaymentRequest, ManualPaymentRequestDto> {
-    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+    private static final DateTimeFormatter settlementDateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
Please follow the Java naming convention
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html